source/plugins/poopmap/index.mjs

```diff
@@ -0,0 +1,40 @@
+//Setup
+export default async function({_login, q, imports, data, _computed, _rest, _graphql, _queries, account}, {enabled = false} = {}) {
+  //Plugin execution
+  try {
+    //Check if plugin is enabled and requirements are met
+    if ((!enabled)||(!q.poopmap))
+      return null
+
+    const {token, days} = imports.metadata.plugins.poopmap.inputs({data, account, q})
+    const {data:{poops}} = await imports.axios.get(`https://api.poopmap.net/api/v1/public_links/${token}`)
+
+    const filteredPoops = poops.filter(poop => {
+      const createdAt = new Date(poop.created_at)
+      const now = new Date().getTime()
+      //Days * hours * minutes * seconds * milliseconds
+      const timeframe = now - (days ?? 7) * 24 * 60 * 60 * 1000
+
+      poop.created_at = createdAt.toString()
+
+      return createdAt.getTime() > timeframe
```

Comment:
I'm pretty sure you can directly do `new Date(poop.created_at) > new Date(Date.now() - days * 24 * 60 * 60 * 1000)` as JS supports natively date comparisons